src/sentry/incidents/tasks.py

```diff
@@ -94,3 +99,22 @@ def build_activity_context(activity, user):
             kwargs={'incident_id': incident.id},
         ),
     }
+
+
+@instrumented_task(
+    name='sentry.incidents.tasks.calculate_incident_suspects',
+    queue='incidents',
+)
+def calculate_incident_suspects(incident_id):
```

Comment:
On the release side of this cache are you thinking of re-computing suspects for all open incidents?

Reply:
@markstory I was just going to recalculate for incidents that are related to the release. So if we have already calculated a suspect commit, and then the commit is removed due to calling `set_commits` on its release I'd recalculate.